includes/definitions/zelax.yaml

```diff
@@ -0,0 +1,20 @@
+os: zelax
+text: 'Zelax OS'
+group: zelax
+type: network
+icon: zelax
+mib_dir: zelax
+over:
+  - { graph: device_bits, text: 'Device Traffic' }
+  - { graph: device_processor, text: 'CPU Usage' }
+  - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+  -
+    sysDescr_regex:
+      - '/^ZES/'
+    sysObjectID:
+      - .1.3.6.1.4.1.7840.2.1
+poller_modules:
```

Comment:
This defaults to false...  If you want something set in your local install, set `lnms config:set os.zelax.discovery_modules.xdsl false`

Please remove these lines.